pkg/odo/cli/component/link.go

```diff
@@ -96,6 +98,7 @@ type LinkOptions struct {
 func NewLinkOptions() *LinkOptions {
 	options := LinkOptions{}
 	options.commonLinkOptions = newCommonLinkOptions()
+	options.commonLinkOptions.csvSupport, _ = util.IsCSVSupported()
```

Comment:
This should actually be done in Complete… New functions should contain as little logic as possible unless it cannot be avoided.